homie-controller/examples/flash.rs

```diff
@@ -0,0 +1,74 @@
+//! Example to turn all Homie devices on and off every 5 seconds, 4 times.
+
+use futures::FutureExt;
+use homie_controller::{Datatype, Event, HomieController, HomieEventLoop, PollError};
+use rumqttc::MqttOptions;
+use std::sync::Arc;
+use std::time::Duration;
+use tokio::task::{self, JoinHandle};
+use tokio::{time, try_join};
+
+fn spawn_poll_loop(
+    mut event_loop: HomieEventLoop,
+    controller: Arc<HomieController>,
+) -> JoinHandle<Result<(), PollError>> {
```

Comment:
This definitely wants to live in the library in some form. Let's wait until we have a few more examples and then sit down and work out what form it should take.